data/config/ndef.json

```diff
@@ -223,19 +223,19 @@
         "options": []
       },
       { 
-        "title": "Species",
+        "title": "Genus Species",
         "field": "species",
         "options": []
       }
     ],
     "fieldMapping" : [
-      { "field": "dataset", "name": "Dataset" },
+      { "field": "dataset", "name": "Resource" },
       { "field": "studyAccession", "name": "Study" },
       { "field": "phenotype", "name": "Phenotype" },
       { "field": "gender", "name": "Gender" },
       { "field": "ethnicity", "name": "Ethnicity" },
       { "field": "strain", "name": "Strain" },
-      { "field": "species", "name": "Genus species" },
+      { "field": "species", "name": "Genus Species" },
```

Comment:
lower case

Reply:
thanks! all fixed in https://github.com/uc-cdis/data-ecosystem-portal/pull/12/commits/81b54b67b622c3dcb4e89d0b448d1f2f8028ae33